### ISSUE TITLE
Iron Golem creation using Iron Golem Assembler

### ISSUE THREAD

**Issue by andris155** [Slimefun/Slimefun4#2425]
## :round_pushpin: Description (REQUIRED)

The Iron Golem creation works from Iron Golem Assembler, and this cause ghost blocks.

## :bookmark_tabs: Steps to reproduce the Issue (REQUIRED)

1. Build Iron Golem from Iron Golem Assembler and Curved Pumpkin.

https://www.youtube.com/watch?v=ay4D5EWaqO0

## :bulb: Expected behavior (REQUIRED)

The Iron Golem creation not work from Iron Golem Assembler.

## :open_file_folder: /error-reports/ Folder

No error report.

## :compass: Environment (REQUIRED)

[16:37:52 INFO]: This Server uses the following setup of Slimefun:
[16:37:52 INFO]: Paper v1_16_R2
[16:37:52 INFO]: CS-CoreLib vDEV - 91 (git e42a05d1)
[16:37:52 INFO]: Slimefun vDEV - 708 (git de5db265)
[16:37:52 INFO]: Metrics build: #17
[16:37:52 INFO]:
[16:37:52 INFO]: Installed Addons: (0)

**Comment by variananora** [comment]
Yep can confirm, but the disappearing Slimefun blocks can be placed again and if you are spamming RMB you can open the last block's GUI for couple seconds and then it's just gone.
Testing video:
https://youtu.be/zjBnR0LFmec
Version:
![image](https://user-images.githubusercontent.com/34732414/104183291-570e5b00-5444-11eb-89ff-5c425d947075.png)

**Comment by JasperChaseTOQ** [comment]
not sure when but this was fixed as i went to test it in slimefun dev version #1027 and it didint work if auto anvil or auto anvil II blocks were in the machine however it worked when both auto anvil and iron blocks were in the machine (regular iron blocks were in the second slot) and it only took the iron blocks not the auto anvils so id say that this issue is resolved
